File: old/contest14/2.cpp
#include "bits/stdc++.h"
/***************************************************/
/***************************************************/
/******************* 3ak3ek ************************/
/***************************************************/
/***************************************************/
using namespace std;
 
typedef long long ll;
typedef long double ld;
typedef pair<int, int> pii;
typedef vector<int> vi;
typedef vector<vi> vvi;
typedef vector<ll> vll;
typedef vector<pii> vpii;
typedef vector<vll> vvll;
typedef pair<ll, ll> pll;
 
#define fastio ios_base::sync_with_stdio(false)
#define debug(x) cerr << #x << " is " << x << "\n"
#define all(v) v.begin(), v.end()
#define rall(v) v.rbegin(), v.rend()
#define For(i, n) for(int i=0; i < n; i++)

void solve(){
	int n;
	cin >> n;
	set<string> s;
	For(i, n){
		string k;
		cin >> k;
		s.insert(k);
	}
	cout << ((s.size() > n/2) ? "YES" : "NO" )<< endl;
}

int main(){
	fastio;
	// freopen("input.in", "r", stdin);
	solve();
	return 0;
}

File: old/contest1/2.py
t = int(input())
for _ in range(t):
    n = int(input())
    if n & 1:
        n -= 1
        if (n//2) & 1:
            print(f"{n//2-2} {n//2+2} 1")
        else:
            print(f"{n//2-1} {n//2+1} 1")
    else:
        print(f"{n//2} {n//2-1} 1")